editor/src/components/inspector/inspector.spec.browser2.tsx

```diff
@@ -444,7 +444,7 @@ export var storyboard = (
         </Storyboard>
       )
       `,
-        ['components.utopia.js']: `import { Card } from './storyboard'
+        ['/utopia/components.utopia.js']: `import { Card } from './storyboard'
```

Comment:
wow, how could this work earlier?

Reply:
it didn't 🤦‍♂️🙈 the tests only asserted what was visible, not what wasn't visible. I'm going back and updating the tests to be more resilient

Reply:
Yeah this is disturbing. I just checked the code and we always check that the filename is prefixed with `/utopia/`, implying that this didn't work

Reply:
I updated the tests in https://github.com/concrete-utopia/utopia/pull/5723/commits/9cc3fecda2031deb6c6f7e485675d6da91fbe0bd